Sources/TuistKit/ProjectEditor/ProjectEditor.swift

```diff
@@ -72,10 +72,13 @@ final class ProjectEditor: ProjectEditing {
             throw ProjectEditorError.noEditableFiles(at)
         }
 
-        let (project, graph) = projectEditorMapper.map(sourceRootPath: destinationDirectory,
+        let (project, graph) = projectEditorMapper.map(sourceRootPath: at,
                                                        manifests: manifests.map { $0.1 },
                                                        helpers: helpers,
                                                        projectDescriptionPath: projectDesciptionPath)
-        return try generator.generateProject(project, graph: graph)
+        return try generator.generateProject(project,
+                                             graph: graph,
+                                             sourceRootPath: project.path,
+                                             xcodeprojPath: nil)
```

Comment:
It looks like the additional parameters aren't being used here - the changes here fix the issue by placing the Manifests project in the project directory (the temporary directory isn't used).

```
ios_app_with_tests/
├── Derived
├── Info.plist
├── Manifests.xcodeproj
├── Playgrounds
├── Project.swift
├── Sources
├── Tests
├── Tests.plist
├── TuistConfig.swift
└── UITests
```

Having a temporary project within the project directory is a valid solution too - perhaps a `.tuist_manifest.xcodeproj`?  + updating the `.gitignore` template to exclude those projects too?

Reply:
My bad. I was so happy that it was working but I didn't realise that it was generated in the same directory as the project.

> Having a temporary project within the project directory is a valid solution too - perhaps a .tuist_manifest.xcodeproj? + updating the .gitignore template to exclude those projects too?

I'd rather not do that because we'd rely on people having their .gitignores well configured. Since this project only matters temporarily while it's being edited, I'd rather generate it in a temporary directory.

I pushed a fix and now it's generated in the right directory. I had to adjust the logic that generates the groups in the project to generate the root relative to the temporary directory.